### ISSUE TITLE
compilation errors

### ISSUE THREAD

**Issue by BugsBunnyBR** [lathonez/clicker#14]
I updated to the latest master version and ran `npm install && ionic build android`...
Are the errors below related to issue  #9 ? If I remove jamine entry from typings.json the jasmine errors 
lines disappear while trying to run the app, however jasmine is needed when running  the tests `rm typings -rf && npm install && npm test`.
I tried to apply [this](http://stackoverflow.com/a/31330272/2742962) but it seems not solve the problem.
Are you able to reproduce this logs?

```
[1m[31mERROR in [default] /home/bunny/git/clicker/node_modules/angular2/src/facade/promise.d.ts:1:9 
Cannot re-export name that is not defined in the module.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/node_modules/ionic-framework/animations/animation.d.ts:48:50 
A 'this' type is available only in a non-static member of a class or interface.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/node_modules/ionic-framework/animations/animation.d.ts:54:50 
A 'this' type is available only in a non-static member of a class or interface.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/es6-shim/es6-shim.d.ts:8:13 
Duplicate identifier 'PropertyKey'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/es6-shim/es6-shim.d.ts:11:4 
Duplicate identifier 'done'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/es6-shim/es6-shim.d.ts:12:4 
Duplicate identifier 'value'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/es6-shim/es6-shim.d.ts:250:4 
Duplicate identifier 'EPSILON'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/es6-shim/es6-shim.d.ts:285:4 
Duplicate identifier 'MAX_SAFE_INTEGER'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/es6-shim/es6-shim.d.ts:292:4 
Duplicate identifier 'MIN_SAFE_INTEGER'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/es6-shim/es6-shim.d.ts:348:4 
Duplicate identifier 'flags'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/es6-shim/es6-shim.d.ts:500:4 
Duplicate identifier 'prototype'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/es6-shim/es6-shim.d.ts:563:4 
Duplicate identifier 'size'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/es6-shim/es6-shim.d.ts:572:4 
Duplicate identifier 'prototype'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/es6-shim/es6-shim.d.ts:583:4 
Duplicate identifier 'size'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/es6-shim/es6-shim.d.ts:592:4 
Duplicate identifier 'prototype'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/es6-shim/es6-shim.d.ts:607:4 
Duplicate identifier 'prototype'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/es6-shim/es6-shim.d.ts:621:4 
Duplicate identifier 'prototype'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:74:8 
Duplicate identifier 'length'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:133:8 
Duplicate identifier 'pass'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:134:8 
Duplicate identifier 'message'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:144:8 
Duplicate identifier 'setTimeout'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:145:8 
Duplicate identifier 'clearTimeout'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:146:8 
Duplicate identifier 'setInterval'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:147:8 
Duplicate identifier 'clearInterval'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:148:8 
Duplicate identifier 'updateInterval'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:150:8 
Duplicate identifier 'currentSpec'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:152:8 
Duplicate identifier 'matchersClass'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:198:8 
Duplicate identifier 'type'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:202:8 
Duplicate identifier 'description'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:204:8 
Duplicate identifier 'totalCount'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:205:8 
Duplicate identifier 'passedCount'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:206:8 
Duplicate identifier 'failedCount'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:208:8 
Duplicate identifier 'skipped'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:218:8 
Duplicate identifier 'values'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:219:8 
Duplicate identifier 'trace'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:223:8 
Duplicate identifier 'matcherName'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:225:8 
Duplicate identifier 'expected'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:226:8 
Duplicate identifier 'actual'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:227:8 
Duplicate identifier 'message'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:228:8 
Duplicate identifier 'trace'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:232:8 
Duplicate identifier 'name'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:233:8 
Duplicate identifier 'message'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:234:8 
Duplicate identifier 'stack'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:257:8 
Duplicate identifier 'env'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:258:8 
Duplicate identifier 'ensured'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:259:8 
Duplicate identifier 'blocks'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:260:8 
Duplicate identifier 'running'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:261:8 
Duplicate identifier 'index'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:262:8 
Duplicate identifier 'offset'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:263:8 
Duplicate identifier 'abort'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:278:8 
Duplicate identifier 'env'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:279:8 
Duplicate identifier 'actual'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:280:8 
Duplicate identifier 'spec'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:281:8 
Duplicate identifier 'isNot'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:303:8 
Duplicate identifier 'not'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:305:8 
Duplicate identifier 'Any'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:344:8 
Duplicate identifier 'id'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:345:8 
Duplicate identifier 'env'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:346:8 
Duplicate identifier 'description'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:347:8 
Duplicate identifier 'queue'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:354:8 
Duplicate identifier 'suite'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:356:8 
Duplicate identifier 'afterCallbacks'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:357:8 
Duplicate identifier 'spies_'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:359:8 
Duplicate identifier 'results_'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:360:8 
Duplicate identifier 'matchersClass'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:385:8 
Duplicate identifier 'id'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:393:8 
Duplicate identifier 'parentSuite'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:416:8 
Duplicate identifier 'identity'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:417:8 
Duplicate identifier 'and'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:418:8 
Duplicate identifier 'calls'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:419:8 
Duplicate identifier 'mostRecentCall'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:420:8 
Duplicate identifier 'argsForCall'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:421:8 
Duplicate identifier 'wasCalled'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:458:8 
Duplicate identifier 'object'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:460:8 
Duplicate identifier 'args'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:473:8 
Duplicate identifier 'started'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:474:8 
Duplicate identifier 'finished'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:475:8 
Duplicate identifier 'result'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:476:8 
Duplicate identifier 'messages'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:490:8 
Duplicate identifier 'Spec'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:491:8 
Duplicate identifier 'clock'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/browser/ambient/jasmine/jasmine.d.ts:492:8 
Duplicate identifier 'util'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/es6-shim/es6-shim.d.ts:8:13 
Duplicate identifier 'PropertyKey'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/es6-shim/es6-shim.d.ts:11:4 
Duplicate identifier 'done'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/es6-shim/es6-shim.d.ts:12:4 
Duplicate identifier 'value'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/es6-shim/es6-shim.d.ts:250:4 
Duplicate identifier 'EPSILON'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/es6-shim/es6-shim.d.ts:285:4 
Duplicate identifier 'MAX_SAFE_INTEGER'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/es6-shim/es6-shim.d.ts:292:4 
Duplicate identifier 'MIN_SAFE_INTEGER'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/es6-shim/es6-shim.d.ts:348:4 
Duplicate identifier 'flags'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/es6-shim/es6-shim.d.ts:500:4 
Duplicate identifier 'prototype'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/es6-shim/es6-shim.d.ts:563:4 
Duplicate identifier 'size'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/es6-shim/es6-shim.d.ts:572:4 
Duplicate identifier 'prototype'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/es6-shim/es6-shim.d.ts:583:4 
Duplicate identifier 'size'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/es6-shim/es6-shim.d.ts:592:4 
Duplicate identifier 'prototype'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/es6-shim/es6-shim.d.ts:607:4 
Duplicate identifier 'prototype'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/es6-shim/es6-shim.d.ts:621:4 
Duplicate identifier 'prototype'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:74:8 
Duplicate identifier 'length'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:75:8 
Duplicate number index signature.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:129:8 
Duplicate string index signature.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:133:8 
Duplicate identifier 'pass'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:134:8 
Duplicate identifier 'message'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:144:8 
Duplicate identifier 'setTimeout'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:145:8 
Duplicate identifier 'clearTimeout'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:146:8 
Duplicate identifier 'setInterval'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:147:8 
Duplicate identifier 'clearInterval'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:148:8 
Duplicate identifier 'updateInterval'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:150:8 
Duplicate identifier 'currentSpec'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:152:8 
Duplicate identifier 'matchersClass'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:198:8 
Duplicate identifier 'type'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:202:8 
Duplicate identifier 'description'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:204:8 
Duplicate identifier 'totalCount'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:205:8 
Duplicate identifier 'passedCount'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:206:8 
Duplicate identifier 'failedCount'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:208:8 
Duplicate identifier 'skipped'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:218:8 
Duplicate identifier 'values'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:219:8 
Duplicate identifier 'trace'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:223:8 
Duplicate identifier 'matcherName'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:225:8 
Duplicate identifier 'expected'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:226:8 
Duplicate identifier 'actual'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:227:8 
Duplicate identifier 'message'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:228:8 
Duplicate identifier 'trace'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:232:8 
Duplicate identifier 'name'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:233:8 
Duplicate identifier 'message'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:234:8 
Duplicate identifier 'stack'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:257:8 
Duplicate identifier 'env'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:258:8 
Duplicate identifier 'ensured'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:259:8 
Duplicate identifier 'blocks'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:260:8 
Duplicate identifier 'running'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:261:8 
Duplicate identifier 'index'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:262:8 
Duplicate identifier 'offset'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:263:8 
Duplicate identifier 'abort'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:278:8 
Duplicate identifier 'env'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:279:8 
Duplicate identifier 'actual'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:280:8 
Duplicate identifier 'spec'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:281:8 
Duplicate identifier 'isNot'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:303:8 
Duplicate identifier 'not'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:305:8 
Duplicate identifier 'Any'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:344:8 
Duplicate identifier 'id'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:345:8 
Duplicate identifier 'env'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:346:8 
Duplicate identifier 'description'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:347:8 
Duplicate identifier 'queue'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:354:8 
Duplicate identifier 'suite'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:356:8 
Duplicate identifier 'afterCallbacks'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:357:8 
Duplicate identifier 'spies_'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:359:8 
Duplicate identifier 'results_'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:360:8 
Duplicate identifier 'matchersClass'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:385:8 
Duplicate identifier 'id'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:393:8 
Duplicate identifier 'parentSuite'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:416:8 
Duplicate identifier 'identity'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:417:8 
Duplicate identifier 'and'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:418:8 
Duplicate identifier 'calls'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:419:8 
Duplicate identifier 'mostRecentCall'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:420:8 
Duplicate identifier 'argsForCall'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:421:8 
Duplicate identifier 'wasCalled'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:458:8 
Duplicate identifier 'object'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:460:8 
Duplicate identifier 'args'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:473:8 
Duplicate identifier 'started'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:474:8 
Duplicate identifier 'finished'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:475:8 
Duplicate identifier 'result'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:476:8 
Duplicate identifier 'messages'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:490:8 
Duplicate identifier 'Spec'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:491:8 
Duplicate identifier 'clock'.[39m[22m

[1m[31mERROR in [default] /home/bunny/git/clicker/typings/main/ambient/jasmine/jasmine.d.ts:492:8 
Duplicate identifier 'util'.[39m[22m

√ Webpack complete

```

**Comment by BugsBunnyBR** [comment]
I think I got it.
````JSON
{
  "compilerOptions": {
    "target": "es5",
    "module": "commonjs",
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true
  },
  "files":[],
  "filesGlob": [
    "./app/**/*.ts",
    "./app/**/*.spec.ts",
    "!./node_modules/**/*.ts",
    "typings/browser.d.ts",
    "typings.main.d.ts"
  ],
  "exclude": [
    "node_modules"
  ],
  "compileOnSave": false,
  "atom": {
    "rewriteTsconfig": false
  }
}
```

**Comment by lathonez** [comment]
I have no issues here. Can you clone from scratch and `npm install`?

**Comment by lathonez** [comment]
FYI you can always see the latest build on travis, which is running everything from scratch on a brand new vm

https://travis-ci.org/lathonez/clicker/builds/111444914

**Comment by BugsBunnyBR** [comment]
rm clicker/ -rf
git clone https://github.com/lathonez/clicker.git
cd clicker/
npm install
ionic build android
explodes in my face :bomb:

**Comment by lathonez** [comment]
Hah, it's the same thing on travis, sorry mate I should have read the issue better, it's a bit early for me! Is this also solved by your PR?

**Comment by BugsBunnyBR** [comment]
No problem, yes, the PR will solve this.

**Comment by lathonez** [comment]
Should be resolved by #17 / 90f677d
